Makefile

```diff
@@ -37,4 +37,5 @@ deploy:
 	go build -o ./cmd/revad/revad ${LDFLAGS} ./cmd/revad 
 	./cmd/revad/revad -c ./cmd/revad/revad.toml -p ./cmd/revad/revad.pid
 deps:
+	sudo apt-get install -y libacl1-dev
```

Comment:
Yay, that way you are breaking any build on systems beside Debian/Ubuntu.

Reply:
:man_shrugging: yep ... acls are platform specific ...

Reply:
But I'm pretty sure we could avoid this cgo dependency

Reply:
@tboerger I hope so ... my google fo fails me and my goling fu is not strong enough